examples/window-app/common/src/WindowApp.cpp

```diff
@@ -40,20 +40,17 @@ void OnTriggerEffect(Identify * identify)
 {
     Clusters::Identify::EffectIdentifierEnum sIdentifyEffect = identify->mCurrentEffectIdentifier;
 
-    ChipLogProgress(Zcl, "IDENTFY  OnTriggerEffect");
-
-    if (identify->mCurrentEffectIdentifier == Clusters::Identify::EffectIdentifierEnum::kChannelChange)
+    if (identify->mEffectVariant != Clusters::Identify::EffectVariantEnum::kDefault)
     {
-        ChipLogProgress(Zcl, "IDENTIFY_EFFECT_IDENTIFIER_CHANNEL_CHANGE - Not supported, use effect varriant %d",
-                        to_underlying(identify->mEffectVariant));
-        sIdentifyEffect = static_cast<Clusters::Identify::EffectIdentifierEnum>(identify->mEffectVariant);
+        ChipLogDetail(AppServer, "Identify Effect Variant unsupported. Using default");
```

Comment:
Cannot we handle this particular use case within its own callback ?

Reply:
I don't understand what do you mean. Which callback? The effect variant is part of the the trigger effects command. It triggers the same callback. Currently in the spec there is only. 1 possible effect variant; default.  This is check is only a mean to present a handling of the effect variant and at least a notification if additional effect variant are added